include/El/core/imports/mpi.hpp

```diff
@@ -34,14 +42,79 @@ namespace mpi {
 #endif
 #endif
 
+// Yes, I realize there's some code duplication here, but it's SO MUCH
+// EASIER for the human to read I really don't care... The
+// preprocessor is the only thing that could care, but it doesn't get
+// feelings. Bwahahaha.
```

Comment:
Just wait for the neural networks to get into the compilers. 🤖

Reply:
YOU'RE GOING TO GIVE THEM FEELINGS??? You monster...

Reply:
Yes, but clang's feelings seem very child-like. So I don't care. :)